presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -213,6 +211,23 @@ Here are the key properties that can be set for a Resource Group:
 * ``jmxExport`` (optional):  If set to ``true``, the statistics of the resource group will be exported via JMX.
   Defaults to ``false``.
 
+* ``perQueryLimits`` (optional): specifies max resources each query in a
+  resource group may consume before being killed. These limits are not inherited from parent groups.
+  May set three types of limits:
+
+    * ``executionTimeLimit`` (optional): Specify an absolute value (i.e. ``1h``)
```

Comment:
```suggestion
    * ``executionTimeLimit`` (optional): Specify an absolute value (for example, ``1h``)
```
This comment applies to the three sub-items in this list - please make the same changes you make to the other two items in the list (``totalMemoryLimit`` and ``cpuTimeLimit``).

* Do not use Latin abbreviations such as i.e.. For context and suggestions, see the GitLab documentation style guide recommended word list entry for [i.e.](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#ie) and [e.g.](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#eg). 

* Please change the indentation of the three items in the sublist to remove the vertical bar. Use the same indentation as the sublist in ``schedulingPolicy`` that begins with ``fair`` and ends with ``query_priority`` (``query_priority`` is visible in the top of the screenshot). 
<img width="792" alt="Screenshot 2024-06-03 at 9 54 27 AM" src="https://github.com/prestodb/presto/assets/7013443/ed6951fb-f52e-4030-8940-f3714d61ec42">